cli/schema/cypress.schema.json

```diff
@@ -181,7 +181,7 @@
         "chromeWebSecurity": {
           "type": "boolean",
           "default": true,
-          "description": "Whether Chrome Web Security for same-origin policy and insecure mixed content is enabled. Read more about this at https://on.cypress.io/web-security"
+          "description": "DEPRECATED: Please use cy.origin() instead. Whether Chrome Web Security for same-origin policy and insecure mixed content is enabled. Read more about this at https://on.cypress.io/web-security"
```

Comment:
We need to add the deprecation to the cypress types as well: https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L2805